addon/editor/input-handlers/backspace-handler.ts

```diff
@@ -397,6 +397,7 @@ export default class BackspaceHandler extends InputHandler {
           },
         })
       );
+      this.rawEditor.model.read();
```

Comment:
shouldn't this be before emitting the content changed event?

Reply:
indeed it should